db/migrate/20160510173147_create_peer_reviews.rb

```diff
@@ -1,13 +1,12 @@
 class CreatePeerReviews < ActiveRecord::Migration
   def change
     create_table :peer_reviews do |t|
+      t.references :result, index: true, foreign_key: true, null: false
       t.integer :reviewer_id, null: false
       t.integer :reviewee_id, null: false
```

Comment:
I tried making references work but couldn't find any examples online or in the documentation of how to have references work with a differently named key.
Am I missing something?